### ISSUE TITLE
Thanks!

### ISSUE THREAD

**Issue by pwilmart** [bittremieux/spectrum_utils#2]
Hi Wout,
Thanks for adding the PTM support and the notebook tutorial. I will see if I can script up some phospho peptide pictures over the weekend. How was the snow? It mostly missed us down here in sunny PDX.
Cheers,
Phil

**Comment by bittremieux** [comment]
The snow kept the university closed for a few days and caused a rush on the grocery stores, but it wasn't actually that bad. 🙂

Adding the modification functionality took a bit longer than I'd hoped because of all the other stuff that I've got going on, but it should work now. I was going to reply to the previous issue with some more information, but the commit automatically closed that issue.

So PTMs should indeed be supported now. You just have to specify the location and the mass difference of each PTM, for example:

```
peptide = 'DLTDYLMK'
modifications = {6: 15.994915}
spec = MsmsSpectrum(..., peptide, modifications)
```

So here the M is oxidated. You can see the full code in the second part of the [notebook](https://github.com/bittremieux/spectrum_utils/blob/master/notebooks/quickstart.ipynb).

Currently modifications aren't supported yet in version 0.1.0 if you install using pip or conda, I need to put out a new release first for that. I'm adding a few more smaller things to include, but normally it should go on PyPI by the end of the week.

Please let me know how it works and if you have any feedback.

**Comment by bittremieux** [comment]
All right, I've published version 0.2.0 on PyPI and Bioconda, so if you install/update via pip or conda you'll get the modification functionality.
